data/korath/korath ships.txt

```diff
@@ -15,58 +15,57 @@ ship "Korath Dredger"
 	attributes
 		category "Heavy Freighter"
 		"cost" 29370000
-		"shields" 39000
-		"hull" 16000
-		"required crew" 201
-		"bunks" 395
-		"mass" 1503
-		"drag" 18.3
-		"heat dissipation" .65
+		"shields" 27000
+		"hull" 22000
+		"required crew" 137
+		"bunks" 331
+		"mass" 1357
+		"drag" 17.1
+		"heat dissipation" .45
```

Comment:
```suggestion
		"shields" 27000
		"hull" 22000
		"required crew" 137
		"bunks" 395
		"mass" 1503
		"drag" 17.1
		"heat dissipation" .45
```

This is quite the dramatic nerf. I haven't tested it out, but this looks like it'd probably feel *more* agile rather than *less* (and yes, I do see the nerf to engine capacity in the next section), which I think is moving the wrong direction. The above suggestion reverts the changes to bunks and mass, but leaves the rest of the changes as I agree with them.

Note regarding drag: Overall, I'm less concerned with max speed than I am with acceleration. As a general rule, I'm fine with big ships being able to go fast, what I *do* object to as a general principle is big ships having high acceleration and especially having high turning. At the root of this, this is more or less in compliance with physics: Technically, everything in the universe (as far as we currently understand it) has the same maximum speed. Some things are capable of vastly higher accelerations, though. I doubt we'll ever have truly accurate physics modelling (I'm not even convinced that it would be good to try), but it does inform my preferences in terms of speed vs acceleration.

In a nutshell, though, I'm fine with drag being reduced arbitrarily; but I very much want to see impacts of the *mass* compared and justified, especially since the next section removes another 7% of its mass in the form of outfit space.

As for the bunks... a) this is the Korath. They pack their people in. b) This ship is used in a ground assault capacity along the lines of the Auxiliary. Aka non-combat in terms of space, but needing large volumes of bunks to hold all those troops. So on the story side there's a need for this to maintain a significantly higher bunk capacity than the Raider (it's already less than double the raider, the proposed change reduces that to a mere 81 more. Especially since the reduction of outfit space in the next part reduces how many bunks can be fit in on top of that.

Reply:
If we're sticking to the current maximum bunks, I think we should probably up the minimum required crew, at least. Right now it's pretty possible to capture a Dredger with a Shield Beetle, when you should probably need something more along the lines of a Kimek Spire/Bactrian/Emerald Sword/Auxiliary. I'm sure the capturing metagame will get shaken up more as we get further along the Exile Expansion, along with Graey's capping rework, but for now it's a bit too easy to obtain for being the second-most powerful capping ship.

Reply:
It may be worth noting that the bunks in the version Zitchas reviewed -- 331 -- were even lower than the bunks put forward by Saugia and Warp-Core -- 347.  The stats they handed me saw a mild reduction in bunks, and a major reduction of crew, which I felt was too much of a buff in terms of free passenger space.

After this review, I pushed a bunk count of 357 - 10 more than Saugia and warp-core, to account for the 10 more crew that I added from their version as well.  See if that fits.

As for the ground assault version, it doesn't actually *use* those bunks mechanically in that mission as far as I can tell.  And, ultimately I'd like to see a proper transport ship serve that role, not a repurposed freighter.

Reply:
> If we're sticking to the current maximum bunks, I think we should probably up the minimum required crew, at least. Right now it's pretty possible to capture a Dredger with a Shield Beetle, when you should probably need something more along the lines of a Kimek Spire/Bactrian/Emerald Sword/Auxiliary. I'm sure the capturing metagame will get shaken up more as we get further along the Exile Expansion, along with Graey's capping rework, but for now it's a bit too easy to obtain for being the second-most powerful capping ship.

I'd just like to note that we do in fact have a built-in means of fixing this exact thing: We can add a crew count alongside the required crew. I use it with my "Ground Assault" variant.
`"crew" 470` will specify that the ship, when spawned by an NPC, will have 470 crew, not just the required 201 (those are the number from the Ground Assault variant currently in-game, not suggested numbers for here.)

So, we could (and in fact would be very fitting for the Korath), to specify `"crew"` attributes on many of their ships that are significantly higher than the minimum required crew. They aren't strictly *required* for the ship to operate, but the Korath house people on their ships, so it makes sense that most of the bunks would be full most of the time.